MyShop/Shop.cs

```diff
@@ -2,30 +2,58 @@
 {
     public class Shop
     {
-        public string shopName;
-        public int openingYear;
-        public int employeeCount;
-        public string city;
+        private string shopName;
+        private int openingYear;
+        private int employeeCount;
+        private string city;
+
+        public string ShopName
```

Comment:
Where there are no logic inside the properties you can use autoproperties and private fields are not required.
Please fix it here and check other classes